.drone/drone.yml

```
@@ -190,6 +190,35 @@ depends_on:
 - docker-amd64
 - docker-arm64
 
+---
+kind: pipeline
+name: cd-to-dev-env
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: image-tag
+  image: alpine/git
+  commands:
+  - apk --update --no-cache add bash
+  - git fetch origin --tags
+  - echo $(./tools/image-tag) > .tags
+
+- image: us.gcr.io/kubernetes-dev/drone/plugins/updater
+  settings:
+    config_json: "{\n  \"destination_branch\": \"master\",\n  \"pull_request_branch_prefix\": \"cd-tempo-dev\",\n  \"pull_request_enabled\": false,\n  \"pull_request_team_reviewers\": [\n    \"tempo-squad\"\n  ],\n  \"repo_name\": \"deployment_tools\",\n  \"update_jsonnet_attribute_configs\": [\n    {\n      \"file_path\": \"ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet\",\n      \"jsonnet_key\": \"tempo\",\n      \"jsonnet_value_file\": \".image-tag\"\n    }\n  ]\n}\n"
```

```suggestion
    config_json: "{\n  \"destination_branch\": \"master\",\n  \"pull_request_branch_prefix\": \"cd-tempo-dev\",\n  \"pull_request_enabled\": false,\n  \"pull_request_team_reviewers\": [\n    \"tempo-squad\"\n  ],\n  \"repo_name\": \"deployment_tools\",\n  \"update_jsonnet_attribute_configs\": [\n    {\n      \"file_path\": \"ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet\",\n      \"jsonnet_key\": \"tempo\",\n      \"jsonnet_value_file\": \".tags\"\n    }\n  ]\n}\n"
```

Just spotted a mistake here, adding here so I remember.